package.json

```diff
@@ -9,7 +9,7 @@
     "Barry Nolte <barry@barrynolte.com>"
   ],
   "engines": {
-    "vscode": "^1.73.0"
+    "vscode": "^1.89.0"
```

Comment:
are you using some feature of the latest vscode? we want the lowest version where it still works, but that's annoying to find so keeping it as is is the second best option.

<img width="712" alt="Screenshot 2024-05-16 at 10 08 52 PM" src="https://github.com/terrastruct/d2-vscode/assets/3120367/c4a7fd46-6914-4c90-86cf-ac8745baf63e">

Reply:
Short answer, no.  Longer answer, I wanted to bump everything up to the lastest versions just in case we hit some sort of regression in functionality with the older vscode library.

So I thought to myself.  Who is the most conservitive, don't want to push the envelope, and most fearful of any backward compatibility issues there is?  The Micrsoft C++ team.  So... what version does their extension support?  1.66.  OK, then, we'll stick with 1.73 where I started, 16 versions ago.